# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/focus/activeelement-after-immediately-focusing-same-site-iframe-contentwindow.html.ini
# Commit: c63ecdad6a3c
# Full Hash: c63ecdad6a3c60ba625517d34bb29c8aae9f5e95
# Author: Henri Sivonen <hsivonen@hsivonen.fi>
# Date: 2021-02-22 17:34:52
# Regressor Bug: 1663931
# File Overlap Count: 1
# Description:
#   Bug 1663931 - Avoid moving focus when changing iframe remoteness. r=nika,mccr8
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D99215
# ==============================================================================

diff -r 828c62bfc0a2 -r c63ecdad6a3c testing/web-platform/meta/focus/activeelement-after-immediately-focusing-same-site-iframe-contentwindow.html.ini
--- a/testing/web-platform/meta/focus/activeelement-after-immediately-focusing-same-site-iframe-contentwindow.html.ini	Mon Feb 22 10:48:53 2021 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,3 +0,0 @@
-[activeelement-after-immediately-focusing-same-site-iframe-contentwindow.html]
-  [Check trailing events]
-    expected: FAIL